### PR TITLE
Update readme to markdown

### PR DESCRIPTION
**Pull request by thisac**
**Context:**
The current readme doesn't work well with PyPI due to the inclusion of raw directives.

**Description of the Change:**
* Changes the format of the readme to markdown instead of rst.

**Benefits:**
PyPI uploads should work.

**Possible Drawbacks:**
None

**Related GitHub Issues:**
None

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -44,6 +44,9 @@
 
 <h3>Documentation</h3>
 
+* Port README to markdown.
```
Comment (josh146) [inline .github/CHANGELOG.md:47]:
```suggestion
* README has been ported to Markdown.
```

File: setup.py [hunk 2]
```diff
@@ -49,7 +49,7 @@
                                           "apps/data/sample_data/*"]},
     "include_package_data": True,
     "description": "Open source library for continuous-variable quantum computation",
-    "long_description": open("README.rst", encoding="utf-8").read(),
+    "long_description": open("README.md", encoding="utf-8").read(),
     "long_description_content_type": "text/x-rst",
```
Comment (josh146) [inline setup.py:53]: I think this line should also be updated, otherwise PyPI will continue to try parsing it as a ReST file

Comment (josh146) [reply setup.py:53]: See https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/ (I think it should be `text/markdown` if I am reading it correctly?)

**Review by josh146 — APPROVED** [review]
Nice, thanks @thisac! Just one line to update in `setup.py`, but apart from that looks good to go

File: setup.py (hunk 2 above)
Comment (thisac) [reply setup.py]: Nice, thanks @josh146. Read through the link and updated it!

**Pull request merged**